x-element.js

```diff
@@ -126,7 +126,7 @@ export default class XElement extends HTMLElement {
   /** Extends HTMLElement.prototype.attributeChangedCallback. */
   attributeChangedCallback(attribute, oldValue, value) {
     const { attributeMap } = XElement.__constructors.get(this.constructor);
-    attributeMap.get(attribute).sync(this, value, oldValue);
+    attributeMap.get(attribute)?.sync(this, value, oldValue);
```

Comment:
it would be good to leave a note here about why we're optional chaining here (else it looks less deliberate and more defensive)

Reply:
Yep. Makes sense 👌